CONTRIBUTING.rst

```diff
@@ -644,7 +644,7 @@ and not packaged together with the core, unless you set ``INSTALL_PROVIDERS_FROM
 variable to ``true``.
 
 In Breeze - which is a development environment, ``INSTALL_PROVIDERS_FROM_SOURCES`` variable is set to true,
-but you can add ``--skip-installing-airflow-providers-from-sources`` flag to Breeze to skip installing providers when
+but you can add ``--install-providers-from-sources=true`` flag to Breeze to skip installing providers when
```

Comment:
Shouldn't it be `True`?

Reply:
Dockls are not yet updated. I leave irt for the switch :)

Reply:
There will be quite many changes in the docs :)